### PR TITLE
OP-132/Sylius upgrade

### PR DESCRIPTION
**Pull request by rafalmal**

### CODE REVIEW

File: tests/Application/config/bundles.php [hunk 1]
```diff
@@ -1,10 +1,17 @@
 <?php
 
+declare(strict_types=1);
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
```
Comment (senghe) [inline tests/Application/config/bundles.php:10]:
```suggestion
/*
 * This file was created by developers working at BitBag
 * Do you need more information about us and what we do? Visit our https://bitbag.io website!
 * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
*/

declare(strict_types=1);

```

File: tests/Application/config/services_test.yaml [hunk 2]
```diff
@@ -4,5 +4,5 @@ imports:
 
 # workaround needed for strange "test.client.history" problem
 # see https://github.com/FriendsOfBehat/SymfonyExtension/issues/88
-services:
-    Symfony\Component\BrowserKit\AbstractBrowser: '@test.client'
+#services:
```
Comment (senghe) [inline tests/Application/config/services_test.yaml:7]: What's this?

File: tests/Application/config/sylius/1.11/bundles.php [hunk 3]
```diff
@@ -1,7 +1,16 @@
 <?php
 
+declare(strict_types=1);
```
Comment (senghe) [inline tests/Application/config/sylius/1.11/bundles.php:3]:
```suggestion
/*
 * This file was created by developers working at BitBag
 * Do you need more information about us and what we do? Visit our https://bitbag.io website!
 * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
*/

declare(strict_types=1);

```

File: tests/Application/config/sylius/1.12/bundles.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
```
Comment (senghe) [inline tests/Application/config/sylius/1.12/bundles.php:10]:
```suggestion
/*
 * This file was created by developers working at BitBag
 * Do you need more information about us and what we do? Visit our https://bitbag.io website!
 * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
*/

declare(strict_types=1);

```

File: tests/Application/public/index.php [hunk 5]
```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
```
Comment (senghe) [inline tests/Application/public/index.php:3]: No OS header

**Review by senghe — CHANGES_REQUESTED** [review]

File: tests/Application/public/index.php (hunk 5 above)
Comment (rafalmal) [reply tests/Application/public/index.php:9]: Fixed in next commit

File: tests/Application/config/sylius/1.11/bundles.php [hunk 6]
```diff
@@ -1,7 +1,23 @@
...
+*/
+
+declare(strict_types=1);
+
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
```
Comment (senghe) [inline tests/Application/config/sylius/1.11/bundles.php:16]:
```suggestion

```

File: tests/Application/config/services_test.yaml [hunk 7]
```diff
@@ -4,5 +4,5 @@ imports:
...
+#    Symfony\Component\BrowserKit\AbstractBrowser: '@test.client'
```
Comment (senghe) [inline tests/Application/config/services_test.yaml:8]:
```suggestion
```

File: tests/Application/config/services_test.yaml (hunk 2 above)
Comment (senghe) [reply tests/Application/config/services_test.yaml]: I've removed this comment

**Pull request merged**